docs-src/cloud/limits.md

```diff
@@ -169,7 +169,42 @@ These operations create Events that are then logged in the Event History.
 The transaction size limit restricts the total size of Events that can be accommodated within a single transaction.
 
 The size limit for any given [Event History](/concepts/what-is-an-event-history) transaction is 4 MB.
-This isn't configurable.
+This limit is non-configurable for Temporal Cloud.
+
+**What is the blob size limit for a transaction?**
+
+- **Blob size limit** for Payloads, including Workflow context and each Workflow and Activity argument and return value:
```

Comment:
I don’t think that documenting warning thresholds is pertinent for Cloud limit docs, as there is no way ATM for customers to see these warnings anyway.